plugins/compliments/plugin/templates/comps.erb

```diff
@@ -0,0 +1,14 @@
+%lh
+<%= "%xc" -%>
+<%= t('compliments.comps_title', :name => char.name.upcase) -%>
+<%= "%xn"  %>
+
+<% paginator.page_items.each do |a| -%>
+<%= "%xcFrom "  -%>
+<%=  a.from -%>
+<%=  right(a.created_at.strftime("%Y-%m-%d"), 65 ) -%>
```

Comment:
You added the date formatter, which is cool, but it's still doing the strftime here.  :)

Reply:
Literally just forgot to save and push this.